docs/Configuration.md

```diff
@@ -83,6 +91,8 @@ configuration variables
 
 * `SCCACHE_DIR` local on disk artifact cache directory
 * `SCCACHE_CACHE_SIZE` maximum size of the local on disk cache i.e. `2G` - default is 10G
+* `SCCACHE_DIRECT` enable direct mode
```

Comment:
Could you add a few more words on what _direct mode_ means?

Reply:
Sure